run-testsuite

```diff
@@ -13,25 +13,21 @@ from os.path import join as joinpath
 class TestFixture(unittest.TestCase):
```

Comment:
Given their nature (integration tests) these tests still don't work because they assume the old way of running mailprocessing (i.e. directly without the setuptools fanciness). Hence they all fail right now (they fail not quite as hard with your changes in place but they still fail). Right now I lean towards removing them entirely because I find the very idea of sinking a lot of time into broken tests revolting (I have to do enough of that during my regular work and thus don't care for it at all in my spare time). Unless you've got a surefire way of running these tests in a way that works, please omit any changes involving `run-testsuite` from your pull request (if you really want to fix them, feel free, but please create a followup pull request in that case).